report/test_analysis.py

```diff
@@ -12,6 +13,11 @@
     batch_dependent_properties,
     reflection_tables_to_batch_dependent_properties,
     combined_table_to_batch_dependent_properties,
+    make_xia2_style_statistics_summary,
```

Comment:
You're screaming "long function names" at me here - could encapsulate them somehow? I appreciate that some of these predate the PR but you're taking over half a VT100 with some of those function names 😱